packages/frontend/src/hooks/organisation/useOrganisation.ts

```diff
@@ -9,8 +9,9 @@ const getOrganisation = async () =>
         body: undefined,
     });
 
-export const useOrganisation = () =>
+export const useOrganisation = (refetchOnMount?: boolean) =>
     useQuery<Organisation, ApiError>({
         queryKey: ['organisation'],
         queryFn: getOrganisation,
+        refetchOnMount,
```

Comment:
to improve this - useQuery wrappers can accept not only `refetchOnMount` but any arbitrary useQuery option. this could improve readability too. (somewhere above there was `useOrganisation(false)` and I had no idea what false was for)